src/backend/opencl/inverse.cpp

```diff
@@ -19,6 +21,10 @@ namespace opencl
 template<typename T>
 Array<T> inverse(const Array<T> &in)
 {
+    if(OpenCLCPUOffload()) {
+        if (in.dims()[0] == in.dims()[1])
+            return cpu::inverse(in);
```

Comment:
When square, the same code as below will need to run on the CPU side. Given CPU is enabled, solve will also run on CPU. So the effect is the same.